src/controller/index.js

```diff
@@ -11,9 +11,9 @@ const compare = (obj1, obj2) => {
 };
 
 class Controller extends ENIP {
-    constructor() {
+    constructor({ queue_max_size } = {}) {
```

Comment:
why pass an object to the constructor in this case? just curious 😄 

perhaps use `constructor(queue_max_size = 200)` instead?

Reply:
In the future, I expect other constructor arguments would be passed based on different 'nice-to-haves' so having it being an object from the start will avoid later migration issues.